<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WidgetTableau</class>
 <widget class="QWidget" name="WidgetTableau">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>629</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WidgetTableau</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>590</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Quitter</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBoxConnexion">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>20</y>
     <width>501</width>
     <height>251</height>
    </rect>
   </property>
   <property name="title">
    <string>Connexion</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Adresse IP</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Nom de la bdd</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>58</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Login</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>130</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Mot de passe</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>34</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>172.18.58.15</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_2">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>61</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>banque</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_3">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>94</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>snir</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_4">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>130</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>snir</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonConnexionBaseDeDonnees">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>40</y>
      <width>191</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Connexion à la base de données</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBoxInfosComptes">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>290</y>
     <width>491</width>
     <height>271</height>
    </rect>
   </property>
   <property name="title">
    <string>Affichage des infos comptes</string>
   </property>
   <widget class="QPushButton" name="pushButtonAfficherInformations">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>40</y>
      <width>281</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Afficher les informations de tous les comptes</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidgetInfos">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>411</width>
      <height>192</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>Nom</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Prénom</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Numéro de compte</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Solde</string>
     </property>
    </column>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>WidgetTableau</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>299</x>
     <y>606</y>
    </hint>
    <hint type="destinationlabel">
     <x>245</x>
     <y>594</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
